src/main/scala/io/iohk/ethereum/keystore/Wallet.scala

```diff
@@ -0,0 +1,9 @@
+package io.iohk.ethereum.keystore
+
+
+import akka.util.ByteString
+import io.iohk.ethereum.domain.{Address, SignedTransaction, Transaction}
+
+case class Wallet(address: Address, privKey: ByteString) {
+  def signTx(tx: Transaction): SignedTransaction = ???
```

Comment:
And this method will be used here. I don't like making `SignedTransaction` dependent on `Wallet` because `Wallet` is a higher-level concept and that should determine the dependency direction.